### ISSUE TITLE
Wed Sites for Soul Again

### ISSUE THREAD

**Issue by anthonyrjwood** [anthonyrjwood/soul_again#24]

**Comment by anthonyrjwood** [comment]
@jaybwstu 

![Image](https://user-images.githubusercontent.com/56207374/187871573-c80fb4a6-6fff-4ee8-ab15-6cf107d66443.png)

Surely, this "mint is live" spinning icon shouldn't be there until the mint is live. We need to update this, so there's a "mint is coming," or something like that.

**Comment by anthonyrjwood** [comment]
@jaybwstu @Hser2bio parts of the site are not loading fully on mobile. Would website compression fix this, or are there modules involved that are not suitable for mobile browsing?

**Comment by jaybwstu** [comment]
@anthonyrjwood should have put this in my website alterations issue. I was thinking of having whitesale live, presale live and mint live

**Comment by anthonyrjwood** [comment]
@jaybwstu yeah, we definitely need a few versions of that icon to switch out when we pass through time frames

**Comment by anthonyrjwood** [comment]
http://soulagain.crypto-elites.club/
http://mint.soulagain.crypto-elites.club/
http://crypto-elites.club/
http://soulagain.crypto-elites.club/coming_soon
